gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureJavaServiceDependencies.java

```diff
@@ -16,8 +16,6 @@
 
 package com.palantir.gradle.conjure;
 
-import com.google.common.collect.Iterables;
```

Comment:
I think this import got swallowed during a merge: [circle](https://app.circleci.com/pipelines/github/palantir/gradle-conjure/1234/workflows/7a881f66-5177-442a-8f39-3f6f33ea31d9/jobs/9610)